framework/docs/src/0_abstract_documentation.md

```diff
@@ -0,0 +1,53 @@
+# Abstract Documentation
+
+Hi and welcome to Abstract, the interchain CosmWasm development platform! Abstract's CosmWasm framework is your gateway 
+to creating amazing things! Whether you're a coding genius or just getting started, Abstract has got your back! By 
+building composable modules you'll be crafting scalable masterpieces in no time!.
+
+## Getting Started
+
+You can read the documentation in the order it is presented, or you can jump to the section that interests you the most! We 
+also have a section with code tutorials that will help you get started with the Abstract SDK if you are more of a hands-on.
+
+- [Abstract SDK Overview](./1_sdk_intro.md)
+- [Tutorials](./10_tutorials.md)
+
+## Explore the Abstract SDK
+
+- [Abstract SDK Overview](./1_sdk_intro.md)
+- [Core Features](./1_sdk_intro.md)
+- [Platform Essentials](./1_sdk_intro.md)
```

Comment:
Are there specific headers that these could link to?

Reply:
I will be creating those later :)